### DISCUSSION TITLE
Install a specific version of colima

### DISCUSSION THREAD

**Discussion by ncharlez** [abiosoft/colima#1007] [General]
I am running Colima on an Intel Mac and version 6.8 has broken my docker compose setup and I can no longer link to external drives. Is there any way install an older version of colima using homebrew? Is there any way to install an older version of colima if homebrew isn't in the equation?

**Comment by amavlyanov** [comment]
You can grab the appropriate version from the latest releases and set it up yourself.

**Reply by molafisoye** [reply]
How??
